content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -884,12 +884,6 @@ enum Role {
 
 ## Defining composite types
 
-<Admonition type="info">
-
-Composite types are available in Preview in versions `3.10.0` and later.
-
-</Admonition>
```

Comment:
This usually gets turned into an introductory sentence that they were first added with `3.10.0` under the `mongodb` preview feature, and is available as GA since 3.12.0.